dashboards-observability/public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -678,10 +684,13 @@ export const Explorer = ({
                           <TimechartHeader
                             dateFormat={'MMM D, YYYY @ HH:mm:ss.SSS'}
                             options={timeIntervalOptions}
-                            onChangeInterval={(intrv) => {
+                            onChangeInterval={(selectedIntrv) => {
+                              const intervalOptionsIndex = timeIntervalOptions.findIndex(item => item.value === selectedIntrv)
+                              const intrv = selectedIntrv.length > 2 ? selectedIntrv.slice(5) : selectedIntrv
```

Comment:
`timeIntervalOptions` values are one character, when would `selectedIntrv.length > 2`?

Reply:
It would be > 2 when the selected option is 'Auto', which has value 'auto_X'